BUILD

```diff
@@ -18,13 +19,31 @@ buildifier(
     mode = "fix",
 )
 
+# Usage:
+#  - bazel run //:requirements.update
 compile_pip_requirements(
     name = "requirements",
     src = "requirements.txt",
     requirements_txt = "requirements_lock.txt",
 )
 
+# Usage:
+#  - bazel run //:venv venv
 py_venv(
     name = "venv",
     deps = all_requirements,
 )
```

Comment:
does this get any of your own python code though, or just your installed dependencies from requirements?

Reply:
just the requirements; it only takes in `all_requirements`. Maybe I could pass it the target of the pybind module... but that's why I added a manual `.pyi` file for the bindings; so my editor could "see" it (still complains the source is missing, but it can do autocompletions from the interface file).

If you meant _all_ of my own Python, no, I expect my editor to gracefully handle live updates to my repo source files; it shouldn't need to be in the venv